### PR TITLE
029_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+	EXPECT_HMIEVENT(event, "UI.GetCapabilities")
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, "UI.GetCapabilities", "SUCCESS", {
+      vrCapabilities = { "TEXT" }, --fake parameter,
+			hmiCapabilities = 
+      {
+				navigation = false,
+				phoneCall = true,
+				steeringWheelLocation = "LEFT"
```
Comment (aderiabin) [inline test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua]: @istoimenova
According test case summary should be trasfered to MobileApp steeringWheelLocation = "CENTER"
In code: steeringWheelLocation = "LEFT"
Please check

Comment (istoimenova) [inline test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua]: @aderiabin, In description is specified: "steeringWeelLocation = LEFT to UI.GetCapabilities"

Comment (istoimenova) [inline test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua]: @aderiabin, done in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/726/commits/38af72c2e9a07fb3d9e08ae035fae1b6ce9cb8af

**Pull request merged**